index.js

```diff
@@ -1,14 +1,41 @@
 'use strict'
 
 var ChangesStream = require('changes-stream')
+var clean = require('normalize-registry-metadata')
+var through2 = require('through2')
 var hypercore = require('hypercore')
 var swarm = require('hyperdrive-archive-swarm')
 var ndjson = require('ndjson')
 var pump = require('pump')
+var afterAll = require('after-all')
 var levelup = require('level')
+var sub = require('subleveldown')
+
+var normalize = through2.obj(function (change, env, cb) {
+  var stream = this
+  var doc = clean(change.doc)
```

Comment:
The `clean()` function will return `undefined` if the module have been deleted. But if a module is deleted after this code have been run the PR will not remove the deleted module retroactively from the db. So it's not really consistent. Consider either keeping deleted modules or removing them retroactively.